src/script/sign.h

```diff
@@ -116,26 +116,26 @@ static constexpr uint8_t PSBT_OUT_BIP32_DERIVATION = 0x02;
 // as a 0 length key which indicates that this is the separator. The separator has no value.
 static constexpr uint8_t PSBT_SEPARATOR = 0x00;
 
-// Takes a stream and multiple arguments and serializes them into a vector and then into the stream
+// Takes a stream and multiple arguments and serializes them as if first serialized into a vector and then into the stream
 // The resulting output into the stream has the total serialized length of all of the objects followed by all objects concatenated with each other.
 template<typename Stream, typename... X>
 void SerializeToVector(Stream& s, const X&... args)
 {
-    std::vector<unsigned char> ret;
-    CVectorWriter ss(SER_NETWORK, PROTOCOL_VERSION, ret, 0);
+    CSizeComputer ss(s.GetType(), s.GetVersion());
```

Comment:
I think this is useful enough to add to add as a function in serialize.h. Making GetSerializeSize() variadic makes sense, but the params would have to be changed around and that means touching a bunch of code.

GetSerializeSizeMany() maybe?

Reply:
Done.